### PR TITLE
Preliminary texture loading support

### PR DESCRIPTION
**Pull request by tracernz**
Loads the albedo textures only. The texture path folder already in the import dialog is used. If this is an absolute path, it is used as-is. If it's a relative path, the standard MSFS layout is assumed (i.e. we look at `<model dir>\..\<texture dir>`). Since Blender doesn't support BC7 DDS files, any DDS filenames are changed to .png (or the .dds suffix is simply dropped in the case of the default aircraft with .png.dds suffix). Error handling is quite weak, so a bit of testing with different models is desirable, and additional handling added to allow the rest of the model to load.

### CODE REVIEW

File: io_msfs_gltf.py [hunk 1]
```diff
@@ -261,11 +263,47 @@ def load_gltf_file(gltf_file_name):
...
+
+            images.append(texture_path.joinpath(uri))
+    return images
+
+
+def get_texture_paths(gltf, texture_path: pathlib.Path, report: Callable):
+    images = get_image_paths(gltf, texture_path, report)
+    textures = []
+    for gltf_tex in gltf['textures']:
+        source = gltf_tex['extensions']['MSFT_texture_dds']['source']
```
Comment (tracernz) [inline io_msfs_gltf.py:286]: This line makes an assumption that the MSFT DDS extension is used for all the textures... that holds for default aircraft but perhaps not third party as the SDK docs say MSFS supports JPG and PNG.

**Comment by AdenFlorian** [comment]
I believe texture importing is already working on the experimental branch

**Comment by tracernz** [comment]
> 
> 
> I believe texture importing is already working on the experimental branch

Ahhhh... I did not notice that branch. Maybe it's best to set the default branch in Github UI to the development one. I have another patch coming that fixes the issues with multiple primitives in some meshes (e.g. A320 wings etc.). Should I raise that against experimental branch instead?

**Comment by tracernz** [comment]
The experimental branch looks like it has a much more ambitious texture importing system. This quick hack got me up and running for easy texture painting at least.

**Pull request closed without merging**